### PR TITLE
initial setup for covid hub dashboard

### PR DESCRIPTION
**Pull request by elray1**

### CODE REVIEW

File: pages/index.qmd [hunk 1]
```diff
@@ -3,23 +3,21 @@ title: "Introduction"
 ---
 
 
+
+This is a dashboard for the [COVID-19 Forecast Hub](https://github.com/CDCgov/covid19-forecast-hub) that is organized by the US CDC. A page with forecast evaluations will be added at a later date.
```
Comment (nickreich) [inline pages/index.qmd:7]: I suggest pointing to the prior COVID-19 Forecast Hub page with an explainer about the new hub, and the launch date, etc... If we don't solve this in this PR, then let's file an issue to make this change.

File: predtimechart-config.yml [hunk 2]
```diff
@@ -66,4 +65,4 @@ task_id_text:
     "69": "Northern Mariana Islands"
     "72": "Puerto Rico"
     "74": "U.S. Minor Outlying Islands"
-    "78": "Virgin Islands"
+    "78": "Virgin Islands"
```
Comment (nickreich) [inline predtimechart-config.yml:68]: noting what looks like maybe a deletion of and endline? probably doesn't matter.

File: site-config.yml [hunk 3]
```diff
@@ -1,6 +1,3 @@
 ---
-hub: "cdcepi/FluSight-forecast-hub/"
-title: "Example Hub Dashboard Website"
-pages:
-  - "about.md"
-
+hub: "CDCgov/covid19-forecast-hub/"
+title: "COVID Hub Dashboard Website"
```
Comment (nickreich) [inline site-config.yml:3]: I suggest "COVID-19 Forecast Hub Dashboard"

"Dashboard Website" is redundant.

File: predtimechart-config.yml [hunk 4]
```diff
@@ -1,12 +1,11 @@
...
 rounds_idx: 0
-model_tasks_idx: 1
+model_tasks_idx: 0
 reference_date_col_name: 'reference_date'
 target_date_col_name: 'target_end_date'
 horizon_col_name: 'horizon'
-initial_checked_models: ['UMass-flusion', 'UMass-trends_ensemble']
-disclaimer: "This is a disclaimer. It presents important information for your stakeholders."
-target_data_file_name: 'target-hospital-admissions.csv'
+initial_checked_models: ['CovidHub-ensemble', 'CovidHub-baseline']
```
Comment (bsweger) [inline predtimechart-config.yml:7]: Checked that the models/target data in lines 7-8 exist over in the corresponding repo 👍

**Review by nickreich — APPROVED** [review]
I have a few minor-ish comments that could be addressed here, ignored or punted to later. Although I'd prefer us to address the one about adding context on the main page about how this hub relates to the other hub.

I didn't try to build this locally, so I didn't verify if it all works.

**Review by bsweger — APPROVED** [review]

**Review by nickreich — APPROVED** [review]
LGTM!

**Pull request merged**